From ba0b742a83f3a494660575545768b3f053f51c25 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Fri, 3 Oct 2014 08:58:40 -0400
Subject: [PATCH] fileio-label: return error when writing fails

The status of actually writing the file was totally ignored.

(cherry picked from commit 754fc0c720eb998b8e47e695c12807ced0ff3602)
---
 src/shared/fileio-label.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/shared/fileio-label.c b/src/shared/fileio-label.c
index 0711826e85..9bfa813b63 100644
--- a/src/shared/fileio-label.c
+++ b/src/shared/fileio-label.c
@@ -33,7 +33,7 @@ int write_string_file_atomic_label(const char *fn, const char *line) {
         if (r  < 0)
                 return r;
 
-        write_string_file_atomic(fn, line);
+        r = write_string_file_atomic(fn, line);
 
         label_context_clear();
 
@@ -47,7 +47,7 @@ int write_env_file_label(const char *fname, char **l) {
         if (r  < 0)
                 return r;
 
-        write_env_file(fname, l);
+        r = write_env_file(fname, l);
 
         label_context_clear();
 
